sgf_wrapper.py

```diff
@@ -160,7 +160,7 @@ def replay_sgf(sgf_contents):
     komi = 0
     if props.get('KM') != None:
         komi = float(sgf_prop(props.get('KM')))
-    result = utils.parse_game_result(sgf_prop(props.get('RE')))
+    result = utils.parse_game_result(sgf_prop(props.get('RE', '')))
```

Comment:
This is needed to replay SGFs without results, it does set the result in the PositionWithContext to '' but I assume that's okay.

Reply:
I don't know what this sentence means.  What does it mean to replay an SGF without a result?  what problem is this working around?

Reply:
any SGF file without a RE tag will throw an error here.
For example a study of opening variations or an ongoing game. I can workaround this by add RE[B+R] to all my files (even though they are only three moves in) but I thought this was nicer. I can back out this change if you don't like the implications.